src/krem.js

```diff
@@ -0,0 +1,7 @@
+export function krem(strings, ...args) {
+  const [namespace, id, ...rest] = args;
```

Comment:
nitpick: `...rest` is not used and could be removed.

Reply:
rest is used on line 4

https://github.com/CanopyTax/kremling/pull/33/files#diff-d5db3c47f7ba6628379679d4f34c30c7R4